src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdClientSettings.Edit.cshtml

```diff
@@ -136,6 +144,79 @@
+                    </a>
+                </td>
+                <td class="text-center"><div class="btn cursor-move"><i class="fas fa-arrows-alt"></i></div></td>
+            </tr>
+        </draggable>
+        <tfoot>
+            <tr>
+                <td class="col-sm-12 text-center" colspan="5">
+                    <a v-on:click="add()" class="btn btn-light w-100 btn-sm"><i class="fas fa-plus small"></i> @T["Add a parameter"]</a>
+                    <input class="form-control" id="@Html.IdFor(m => m.Parameters)" name="@Html.NameFor(m => m.Parameters)" type="hidden" v-bind:value="getParametersFormattedList()" />
+                </td>
+            </tr>
+        </tfoot>
+    </table>
+</script>
+
+<script type="text/x-template" id="parameters-modal">
+    <div class="modal fade @Html.IdFor(m => m)-ModalBody text-left" role="dialog" aria-hidden="true">
+        <div class="modal-dialog modal-dialog-centered" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title">@T["Edit Data"]</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <div class="form-group">
+                        <label for="parameters-textarea">@T["Parameters"]</label>
+                        <textarea id="parameters-textarea" name="parameters-textarea" rows="8" class="form-control" v-bind:value="JSON.stringify(data.parameters)" v-on:input="data.parameters = JSON.parse($event.target.value)"></textarea>
+                        <span class="hint">@T["A JSON representation of the allowed values, e.g. {0}", "[ { name: 'First parameter', value: 'parameter1' }, { name: 'Second parameter', value: 'parameter2' } ]"]</span>
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-primary btn-submit" v-on:click="closeModal()">@T["OK"]</button>
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">@T["Cancel"]</button>
+                </div>
+            </div>
+        </div>
+    </div>
+</script>
+
+<div id="@Html.IdFor(m => m)" class="field-editor field-editor-parameters">
+    <div class="row mb-3">
+        <div class="col-12 col-lg-6">
+            <label asp-for="Parameters">@T["Parameters"]</label>
+            <a href="javascript:void(0)" v-on:click="showModal" class="float-right" title="@T["Edit Data"]"><i class="fas fa-edit"></i></a>
+            <parameters-table :data="sharedState"></parameters-table>
+            <parameters-modal :data="sharedState" v-on:reload-data="reloadData()"></parameters-modal>
+            <div class="hint">@T["Optional parameters that can sent with the authorize request."]</div>
```

Comment:
```suggestion
            <div class="hint">@T["Optional parameters that can be sent with the authorize request."]</div>
```

But not sure, maybe it is allowed in english ;)

Reply:
No, just a bad English sentence from me ;)